packages/grafana-ui/src/options/models.gen.ts

```diff
@@ -1,5 +1,6 @@
 // TODO: this should be generated with cue
 import { VizLegendOptions, VizTooltipOptions } from '../components';
+import { CandlestickFieldMappings } from '../../../../public/app/plugins/panel/candlestick/types';
```

Comment:
:grimacing:  I think we can remove all this right?   does it need to be here now?